code/go/0chain.net/blobber/flags.go

```diff
@@ -32,12 +33,14 @@ func init() {
 	flag.StringVar(&metadataDB, "db_dir", "", "db_dir")
 	flag.StringVar(&logDir, "log_dir", "", "log_dir")
 	flag.IntVar(&httpPort, "port", 0, "port")
-	flag.IntVar(&httpsPort, "https-port", 0, "https-port")
-	flag.StringVar(&httpsCertFile, "https-cert-file", "", "https-cert-file")
-	flag.StringVar(&httpsKeyFile, "https-key-file", "", "https-key-file")
+	flag.IntVar(&httpsPort, "https_port", 0, "https_port")
+	flag.StringVar(&httpsCertFile, "https_cert_file", "", "https_cert_file")
+	flag.StringVar(&httpsKeyFile, "https_key_file", "", "https_key_file")
 	flag.StringVar(&hostname, "hostname", "", "hostname")
 	flag.StringVar(&configDir, "config_dir", "./config", "config_dir")
 
+	flag.StringVar(&hostUrl, "hosturl", "", "register url on blockchain instead of [schema://hostname+port] if it has value")
```

Comment:
Can I provide both hosturl as well as hostname? It is contradicting to have hostname with port as well as hosturl, do we have a use-case for hosturl?

Reply:
the url is concat by `http(s)://hostname:port`. but it is https://dev.0chain.net/blobber01

so it doesn't work with --hostname https://dev.0chain.net/blobber01. We don't know which pattern should be used. 

because hostname is not use-case, so just keep `hosturl` as `hostname`